commit 93f9d2728a901c1d6bc574a15853bf335c18b7dd (refs/changes/03/3803/3)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-04-11T14:00:30-07:00 (1 year, 6 months ago)
    
    OS-6887 test-vm "test log archiving" failure: new console.log and platform.log surprised test case
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/src/vm/tests/test-docker.js b/src/vm/tests/test-docker.js
index 8f2af6dd..19050933 100644
--- a/src/vm/tests/test-docker.js
+++ b/src/vm/tests/test-docker.js
@@ -1,4 +1,4 @@
-// Copyright 2017 Joyent, Inc.  All rights reserved.
+// Copyright (c) 2018, Joyent, Inc.
 //
 // These tests ensure that docker flag works as expected when setting/unsetting
 // Also test that /etc/resolv.conf, /etc/hosts and /etc/hostname are set
@@ -1080,8 +1080,10 @@ test('test log archiving', function (t) {
         }, function (cb) {
             var log_dir = '/zones/' + state.uuid + '/logs';
 
-            // ensure we have 2 log files now
+            // Ensure we have 2 "stdio.log*" files now.
             fs.readdir(log_dir, function (err, files) {
+                files = files.filter(
+                    function (f) { return f.indexOf('stdio.log') === 0; });
                 t.ok(files.length === 2, 'vm logs: ' + JSON.stringify(files));
                 cb();
             });
